lib/elrondex/sc/pair_sc.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Elrondex.Sc.PairSc do
   alias Elrondex.{Sc, Transaction, Account, REST, ESDT, Pair, Network}
 
+  @doc """
+  Swaps token1 amount for a minimum of token2 amount.
+
+  ## Arguments
+    * `account` - An elrond account (as described in lib/account)
```

Comment:
There might be a better explication for what 'account' does eg: "the account that does the pair"

Also, I don't find the usage of the ()